src/data/dataMethods.ts

```diff
@@ -367,11 +370,32 @@ export function _create<R extends Record<string, Any>>(
   return _dataRequest(client, httpRequest, 'mutate', {mutations: [mutation]}, opts)
 }
 
+const hasDataConfig = (client: Client) =>
```

Comment:
Nice! This is super nit, feel free to ignore: could I get a newline between each function?

Reply:
Snorre ~Eskeland~ Linter Brekke!